Assets/Scripts/DefaultFileBeadsProvider.cs

```diff
@@ -1,49 +1,50 @@
+using System;
 using System.Collections.Generic;
 using UnityEngine;
 
-public class DefaultFileBeadsProvider: ILinkBeadsProvider
+public class DefaultFileBeadsProvider : ILinkBeadsProvider
 {
-        private readonly TextAsset _linkPointsTextFile;
+    private readonly TextAsset _linkPointsTextFile;
 
-        public DefaultFileBeadsProvider(int crossingNumber, int ordering, int numComponents = 1)
-        {
-                _linkPointsTextFile = DefaultFileAccessor.GetTextAsset(crossingNumber, ordering, numComponents);                
-        }
+    public DefaultFileBeadsProvider(int crossingNumber, int ordering, int numComponents = 1)
+    {
+        _linkPointsTextFile = DefaultFileAccessor.GetTextAsset(crossingNumber, ordering, numComponents);
```

Comment:
This may return null, is that a problem?

Reply:
I'm leaving it as is for now because this PR has been out for a long time and we can address that later.